addon/-private/system/references/record.js

```diff
@@ -11,28 +18,140 @@ RecordReference.prototype = Object.create(Reference.prototype);
 RecordReference.prototype.constructor = RecordReference;
 RecordReference.prototype._super$constructor = Reference;
 
+/**
+   The `id` of the record that this reference refers to.
+
+   Together, the `type` and `id` properties form a composite key for
+   the identity map.
+
+   Example
+
+   ```javascript
+   var userRef = store.getReference('user', 1);
+
+   userRef.id(); // '1'
+   ```
+
+   @method id
+   @return {String} The id of the record in this belongsTo relationship.
+*/
 RecordReference.prototype.id = function() {
   return this._id;
 };
 
+/**
+   How the reference will be looked up with it is loaded:
+
+   * `link`, a URL
```

Comment:
Hmm, I believe that this is not supported at the moment. At least I am not aware of this... What am I missing?